### PR TITLE
Limiting transmission batches to 25

### PR DESCRIPTION
**Pull request by sree-cfa**

### CODE REVIEW

File: crontab [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Run weekly on Wednesday at Noon PDT
-# 0 19 * * WED java -Dserver.port=9191 -jar /opt/la-doc-uploader/app.jar transmit
+# 0 19 * * WED java -Dserver.port=9191 -jar /opt/form-flow-starter-app/app.jar transmit
 
 # Run every hour
-# @hourly java -Dserver.port=9191 -jar /opt/la-doc-uploader/app.jar transferSubmissions
+@hourly java -Dserver.port=9191 -jar /opt/form-flow-starter-app/app.jar transferSubmissions
```
Comment (analoo) [inline crontab:5]: @sree-cfa - curious: why is this referencing the form-flow starter app instead of the la-doc-uploader?

Comment (sree-cfa) [reply crontab:5]: This is just the path we've been using in the Dockerfile - the "MKDIR" part

File: src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java [hunk 2]
```diff
@@ -77,6 +76,33 @@ public SubmissionTransfer(SubmissionRepository submissionRepository, Transmissio
...
+    List<Submission> queuedSubmissions = transmissionRepository.submissionsToTransmit(Sort.unsorted(), TransmissionType.SNAP);
+    if (queuedSubmissions.isEmpty()) {
+      log.info("Nothing to transmit");
+      return;
+    }
+    log.info("Found %s queued transmissions".formatted(queuedSubmissions.size()));
+
+    queuedSubmissions = queuedSubmissions.stream()
+        .filter(submission -> (submission.getSubmittedAt().isBefore(submittedAtCutoff))).toList();
+    log.info("Excluding %s submitted within last 2 hours".formatted(queuedSubmissions.size()));
```
Comment (spokenbird) [inline src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java:90]: For this log wouldn't queued submissions size be the number of applications that **are** in the window where they should be sent (as they are older than 2 hours ago?).

Comment (sree-cfa) [reply src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java:90]: Oh yeah youre right

**Review by spokenbird — APPROVED** [review]

**Pull request merged**